sdk/src/test/scala/io/horizen/account/websocket/WebSocketAccountServerEndpointTest.scala

```diff
@@ -106,6 +114,23 @@ class WebSocketAccountServerEndpointTest extends JUnitSuite with MockitoSugar wi
     null
   }
 
+  @Test
+  def allowedOriginsTest(): Unit = {
+    server = WebSocketAccountServerRef(mockedSidechainNodeViewHolderRef, rpcProcessor, mockedWebsocketServerSettingsWithExternalOrigin)
```

Comment:
let's have 2 different tests:
1) for `mockedWebsocketServerSettings` when we have a default behavior (accept localhost only)
2) for `mockedWebsocketServerSettingsWithExternalOrigin` when we expect only remote one `185.123.0.12`, so localhost connection is refused.